src/widget/capslockindicator.cpp

```diff
@@ -2,20 +2,27 @@
 #ifdef QTOX_PLATFORM_EXT
 #include "src/platform/capslock.h"
 #endif
+#include <QDebug>
 
 CapsLockIndicator::CapsLockIndicator(QWidget *parent) : QToolButton(parent)
 {
-    inputSize = QSize(130, 23);
     cleanInputStyle = parentWidget()->styleSheet();
 
     QIcon icon = QIcon(":img/caps_lock.svg");
     setIcon(icon);
-    QSize iconSize(inputSize.height(), inputSize.height());
-    setIconSize(iconSize);
     setCursor(Qt::ArrowCursor);
-    move(inputSize.width() - inputSize.height(), 0);
     setStyleSheet("border: none; padding: 0; color: white");
     setToolTip(tr("CAPS-LOCK ENABLED"));
+    updateSize();
+}
+
+void CapsLockIndicator::updateSize() {
```

Comment:
If you are using qt-creator you can set the coding style to "Qt" which is what we are using.